content/docs/introducing-jsx.md

```diff
@@ -1,36 +1,36 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
-permalink: docs/introducing-jsx.html
+title: Pagpapakilala sa JSX
+permalink: docs/pagpapakilala-sa-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Mangyaring tingnan ang variable declaration na ito:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Ang syntax na ito ay hindi isang string o HTML.
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+Ito ay tinatawag na JSX, at isa itong halimbawa ng syntax extension sa JavaScript. Inirerekomenda naming gamitin ito kasama ng React para bigyan ng porma ang UI. Maihahalintulad ang JSX sa isang template language, ngunit maaari rin itong ipaghalo sa iba pang JavaScript na code.
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+Mula sa JSX, makabubuo tayo ng mga React "element". Ipapakita sa [susunod na paksa](/docs/rendering-elements.html) ang pag-render ng JSX papunta sa DOM. Samantala, nasa sumusunod na bahagi ang mga halimbawa ng JSX para masimulan ang paggamit nito.
```

Comment:
section = sa approved PR https://github.com/reactjs/tl.reactjs.org/pull/3/files ginamit yung word na kabanata. Pwede papalit ng `paksa` to `kabanata` para consistent.